api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -763,9 +765,26 @@ public User setUserActivationKey(User user) {
 		LoginCredential credentials = dao.getLoginCredential(user);
 		credentials.setActivationKey(activationKey);	
 		dao.setUserActivationKey(credentials);	
-			
-		//Send Email with unhashed  activation key and Date:
 		
+		String link = Context.getAdministrationService().getGlobalProperty(OpenmrsConstants.GP_HOST_URL) + token;
+		String msg = "Hi " + user.getUsername()
+		        + ", \n It looks like you like to change your password, click on the following link to do so.\n" + link
```

Comment:
Somewhat a lot of repetition of **like**, can you rephrase this please?

Reply:
Do line breaks really work here? I would think html would

Reply:
For a start, add a file to the classpath containing this message and read in the contents when sending the email, in the feature it will be a template that can support localization too

Reply:
@wluyima Line breaks work instead of html <br /> tags. Please am having issues adding txt files to classpath.